.env

```
@@ -1,8 +1,8 @@
-PG_HOST=aws-0-us-east-1.pooler.supabase.com
+PG_HOST=aws-0-us-east-2.pooler.supabase.com
 PG_PORT=6543
 PG_DATABASE=postgres
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The PG_DATABASE key should go before the PG_HOST key

(UnorderedKey)